SoftLayer/managers/ticket.py

```diff
@@ -80,6 +80,9 @@ def update_ticket(self, ticket_id=None, body=None):
         """
         return self.ticket.addUpdate({'entry': body}, id=ticket_id)
 
+    def upload_attachment(self, ticket_id=None, file=None):
```

Comment:
I agree with having `file_name` and the `file_contents` as different parameters. Also, ticket_id is required, so it seems like it should be a positional argument.